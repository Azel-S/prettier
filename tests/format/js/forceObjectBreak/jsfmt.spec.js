run_spec(__dirname, ["babel", "flow", "typescript"], { forceObjectBreak: "preserve" });
run_spec(__dirname, ["babel", "flow", "typescript"], { forceObjectBreak: "forceMultiLine" });
run_spec(__dirname, ["babel", "flow", "typescript"], { forceObjectBreak: "forceSingleLine" });