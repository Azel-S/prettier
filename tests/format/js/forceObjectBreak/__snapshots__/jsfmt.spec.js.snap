// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`forceObjectBreak.ts - {"forceObjectBreak":"forceMultiLine"} format 1`] = `
====================================options=====================================
forceObjectBreak: "forceMultiLine"
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
const obj = { foo: 1, bar: 2};

=====================================output=====================================
const obj = {
  foo: 1,
  bar: 2,
};

================================================================================
`;

exports[`forceObjectBreak.ts - {"forceObjectBreak":"forceSingleLine"} format 1`] = `
====================================options=====================================
forceObjectBreak: "forceSingleLine"
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
const obj = { foo: 1, bar: 2};

=====================================output=====================================
const obj = { foo: 1, bar: 2 };

================================================================================
`;

exports[`forceObjectBreak.ts - {"forceObjectBreak":"preserve"} format 1`] = `
====================================options=====================================
forceObjectBreak: "preserve"
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
const obj = { foo: 1, bar: 2};

=====================================output=====================================
const obj = { foo: 1, bar: 2 };

================================================================================
`;
